monai/apps/mmars.py

```diff
@@ -0,0 +1,154 @@
+# Copyright 2020 - 2021 MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import warnings
+from typing import Mapping
+
+import torch
+
+import monai.networks.nets as monai_nets
+from monai.apps.utils import download_and_extract
+
+__all__ = ["download_mmar", "load_from_mmar", "MODEL_DESC"]
+
+
+MODEL_DESC = (
+    {
+        "id": "clara_pt_prostate_mri_segmentation_1",
+        "name": "clara_pt_prostate_mri_segmentation",
+        "url": "https://api.ngc.nvidia.com/v2/models/nvidia/med/clara_pt_prostate_mri_segmentation/versions/1/zip",
+        "doc": "https://ngc.nvidia.com/catalog/models/nvidia:med:clara_pt_prostate_mri_segmentation",
+        "file_type": "zip",
+        "hash_type": "md5",
+        "hash_val": None,
+        "model_file": os.path.join("models", "model.pt"),
+    },
+    {
+        "id": "clara_pt_covid19_ct_lesion_segmentation_1",
+        "name": "clara_pt_covid19_ct_lesion_segmentation",
+        "url": "https://api.ngc.nvidia.com/v2/models/nvidia/med/clara_pt_covid19_ct_lesion_segmentation/versions/1/zip",
+        "doc": "https://ngc.nvidia.com/catalog/models/nvidia:med:clara_pt_covid19_ct_lesion_segmentation",
+        "file_type": "zip",
+        "hash_type": "md5",
+        "hash_val": None,
+        "model_file": os.path.join("models", "model.pt"),
+    },
+)
+"""
+Collection of the remote MMAR descriptors
+"""
+
+
+def _get_model_spec(idx):
```

Comment:
I think maybe it's better to define all the items in a JSON config file instead of hard-code in the source code, we may frequently update the items in Clara side.

Reply:
thanks, I put it into a separate `.py` file, I feel it'll benefit from our python code formatting, I think we can provide json loader for external config if needed.